packages/wallet/src/Transaction/ensureValidityInterval.ts

```diff
@@ -1,8 +1,20 @@
-import { Cardano } from '@cardano-sdk/core';
+import { Cardano, Seconds } from '@cardano-sdk/core';
+
+const twoHours = Seconds(2 * 3600);
+
+export const calcTimeOffsetSlotNumber = (
```

Comment:
Missing a docblock

Reply:
done: [fixup! feat(wallet): convert static TTL to be based on slotLength](https://github.com/input-output-hk/cardano-js-sdk/pull/576/commits/0e41ddffc1829232e46a91fc0ef2a8a74a3c7a6f)